front/src/config/i18n/fr.json

```diff
@@ -791,6 +791,18 @@
         "magnet": "Capteur magnétique"
       }
     },
+    "oauth2": {
+      "buttonConnect": "Connecter maintenant",
+      "apiKeyLabel": "Saisissez ensuite votre clé client_id et secret_id ici:",
+      "errorEmptyConfig": "Client Id et Secret Id ne doivent pas être vide !",
+      "errorServiceId": "Service Id de l'intégration courante n'a pas été trouvé !",
+      "errorAuthorizationUri": "Erreur dans la configuration du provider pour l'autorisation de connexion !",
+      "clientId": "Client id du compte",
+      "secretId": "Secret id du compte",
+      "instructions": "À cette étape, veuillez remplir les champs client_id et secret_id ci-dessous et cliquez sur le bouton de connexion.",
```

Comment:
Please prefer verbose instead of client_id and secret_id (client secret).

Reply:
i fix it